TikTokApi/browser.py

```diff
@@ -203,7 +207,7 @@ def sign_url(self, **kwargs):
     def clean_up(self):
         try:
             self.browser.close()
-        except:
+        except BaseException:
```

Comment:
Ooh, this is dangerous.  You should never need to handle `BaseException` explicitly.  The reason is that python handles signals (e.g. `SIGTERM` etc) as Exceptions.  But those exception classes don't inherit from `Exception`, they inherit from `BaseException`. So this would also trigger if a user hit cmd+c (or ctrl+c) at the wrong moment.  This is the same reason you should never use a bare `except`.

If you want to handle 'all unexpected exceptions', then `except Exception` is the way to go.
   
```suggestion
        except Exception:
```